src/Service/Fleets.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+namespace Platformsh\Cli\Service;
+
+use Platformsh\Cli\Local\LocalProject;
+
+
+class Fleets
+{
+
+    const FLEET_DOES_NOT_EXIST = 0;
```

Comment:
I'd prefer exceptions rather than error codes - getFleet() perhaps could return either a fleet object or false, but addFleet() and removeFleet() should just be relied on for their named purpose, not worrying about the file not existing etc.

(happy to rejig this myself)

Reply:
The case where a fleet does not exist is not an Exception, it's just incorrect input data, nor is it sufficiently complex to warrant the kind of cascade that Exceptions are useful for. 

So, we can do this, but it will be a lot more work for the same result. To make this functionally equivalent we'd need an Exception class for each case, then try/catch handling for those cases, which would just get us back to this point, since there is no further Exception handling required as we don't through any other kinds.

Reply:
IMO we wouldn't need a separate exception class or try/catch.. I'll propose something in a branch.